themes/default/layouts/page/pricing.html

```diff
@@ -322,6 +322,25 @@ <h4><a href="#faq-billing" class="text-gray-500">Billing and Support</a></h4>
 
         <p class="text-2xl font-bold lg:hidden mt-12 pl-5">Product</p>
         <ul class="list-none p-0 mx-5 xl:ml-12 lg:hidden" data-faq-type="faq-product">
+            <li class="accordion-item text-2xl py-3">
+                {{ partial "accordian-header" (dict "text" "What are Pulumi open source and Pulumi Service?" "large_header" true) }}
+
+                <div class="accordion-item-body-no-animation text-base">
+                    <p>
+                         Pulumi's modern Infrastructure as Code CLI and SDK are an open-source project that
+                         is supported by an active community. Pulumi Service is a managed service for the
+                         open source CLI and SDK. It tracks your infrastructure’s state and coordinates
+                         updates with the CLI, which creates or updates resources to reach your infrastructure’s
+                         desired state. It also manages secrets, supports SAML SSO, integrates with CI/CD pipelines,
+                         enforces compliance rules, and much more.
+                    </p>
+                    <p>
+                        You're not required to use <a class="underline" href="{{ relref . "/product/pulumi-service" }}">Pulumi Service</a>.
+                        You can use any cloud or on-premises storage to build and run your own backend.
```

Comment:
@zchase could you please link "run your own backend" to this?
https://www.pulumi.com/docs/intro/concepts/state/